azure-pipelines.yml

```diff
@@ -1,8 +1,18 @@
+strategy:
+  matrix:
+    'Node 8': 
+      versionSpec: '8.x'
```

Comment:
Please add tests for node 6 also - since currently it is still used as execution handler